mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/English.kt

```diff
@@ -184,3 +185,15 @@ object English : LocalizedStrings {
   override val tournamentsCreateElimDemomN = { denominator: Int -> "1 / $denominator" }
   override val tournamentsCreateElimDepthFinal = "Final"
 }
+
+fun Duration.toDurationString(): String {
```

Comment:
Missing some documentation, and this must be private. Also, I'd rename it to `toEnglishString`, to avoid potential name clashes when new languages are added.